code/modules/loadout/loadout_role_restricted.dm

```diff
@@ -906,7 +906,6 @@
 /datum/loadout_entry/restricted/engineering/uniform/eng_skirt_pleated
 	name = "Engineers Pleated Skirt"
 	path = /obj/item/clothing/under/rank/engineer/skirt_pleated
-	allowed_roles = list("Chief Engineer", "Atmospheric Techician")
```

Comment:
please dont remove the allowed roles and keep it in restricted, either move it to the unrestricted section, or change the line to allowed_roles = list("Chief Engineer", "Atmospheric Technician", "Station Engineer", "Senior Engineer")